plugins/plugin-docker/che-plugin-openshift-client/pom.xml

```diff
@@ -42,6 +42,7 @@
         <dependency>
             <groupId>io.fabric8</groupId>
             <artifactId>kubernetes-client</artifactId>
+            <version>2.1.1</version>
```

Comment:
Are you sure this is approved version of client?
take a look https://github.com/eclipse/che-dependencies/blob/master/pom.xml#L53-L54